e2e/nomostest/clusters/gke.go

```diff
@@ -109,8 +113,14 @@ func waitOperation(ctx context.Context, t testing.NTB, operation string) error {
 }
 
 func listAndWaitForOperations(ctx context.Context, t testing.NTB, name string) error {
-	ctxWithTimeout, cancel := context.WithTimeout(ctx, 5*time.Minute)
-	defer cancel()
+	var ctxWithTimeout context.Context
```

Comment:
you could just replace the ctx var. that would avoid the else branch

Reply:
done